extensions/sql-database-projects/package.json

```diff
@@ -247,6 +265,16 @@
         {
           "command": "sqlDatabaseProjects.newScript",
           "when": "view == sqlDatabaseProjectsView && viewItem == databaseProject.itemType.project || viewItem == databaseProject.itemType.folder",
+          "group": "3_dbProjects_newItem@7"
+        },
+        {
+          "command": "sqlDatabaseProjects.newPreDeployScript",
```

Comment:
I feel it shouldn't be a different group, since this falls under the bucket of "NewItem" (3_dbProjects_newItem), and also because we don't really stop customers from adding more than one pre or post deployment script.

Reply:
But I am open for discussion on this.

Reply:
Okay, as discussed, let's keep with the current behavior to match SSDT in this case.  I made https://github.com/microsoft/azuredatastudio/issues/11955 to track the ability to select the active pre/post-deploy script.